_resources/port1.0/compilers/clang_compilers.tcl

```diff
@@ -4,19 +4,34 @@
 # clang_dependency PortGroup, add it to any new dependencies of the
 # new version, and update the blacklisting in the other clang ports.
 
-# does Clang work on all i386 and x86_64 systems?
-# according to https://packages.macports.org/clang-7.0/,
-# clang builds back to Mac OS X 10.6
-lappend compilers macports-clang-9.0 \
-                  macports-clang-8.0 \
-                  macports-clang-7.0 \
-                  macports-clang-6.0 \
-                  macports-clang-5.0
-if {${os.major} < 16} {
+# no macports clang compilers are available for PowerPC
+# macports-clang-9.0 and less are available for darwin10+
+# macports-clang-7.0 and less are available for darwin9+
+# macports-clang-3.4 and less are available for darwin8+
```

Comment:
The comment is not quite right, as clang 3.x are only available for Darwin < 16.
Also, clang 3.7 is not available on Darwin 9.

Reply:
clang-3.7 works like greased lightening on Darwin 9. It's a prime compiler there. Also, there is a comment lower down about 10.6, so .... ....